dashboard/config/routes.rb

```diff
@@ -96,8 +96,8 @@ def redirect_to_teacher_dashboard
         get '/' + key.to_s + '/:channel_id/edit', to: 'projects#edit', key: key, as: key.to_s + 'project_edit'
         get '/' + key.to_s + '/:channel_id/view', to: 'projects#show', key: key, as: key.to_s + 'project_view', readonly: true
       end
+      get '/angular', to: 'projects#angular'
```

Comment:
Should be equivalent to https://studio.code.org/p/projects.  @laurelfan knows more about this view but I think it gets loaded by Angular on the `/projects` overview.